zenodo/modules/deposit/templates/zenodo_deposit/edit.html

```diff
@@ -135,6 +135,45 @@
           };
         }]);
 
+        angular.module('invenioRecords.directives')
+        .directive('extractButton',['$rootScope', 'InvenioRecordsAPI', function($rootScope, InvenioRecordsAPI) {
+
+          function link($scope, elem, attrs, vm) {
+            $scope.extractMetadata = function(f) {
+                var url = `/record/extractmetadata/${ f.version_id }`
```

Comment:
This works for the prototyping (since we are all using modern browsers) but will need to be changed for production. Unfortunately until we improve our JS build process through Webpack, we will have to be checking [Can I Use...](https://caniuse.com/#feat=template-literals) before being adventurous :wink: